build.sbt

```diff
@@ -12,6 +12,9 @@ import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin._
 import uk.gov.hmrc.versioning.SbtGitVersioning
 
 import scala.language.postfixOps
+import com.typesafe.sbt.packager.MappingsHelper._
+
```

Comment:
I would add a comment here along the lines of the slack reply you gave ie `my understanding is publishing processed changed when we moved to the open and now it is done in production mode (was in dev previously). hence, we encounter the problem accessing "public" folder`